integration-tests/spring-data-jpa/pom.xml

```diff
@@ -68,6 +74,36 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-invoker-plugin</artifactId>
+                <version>3.1.0</version>
+                <executions>
+                    <execution>
+                        <id>integration-tests</id>
+                        <goals>
+                            <goal>install</goal>
+                            <goal>run</goal>
+                            <goal>verify</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <cloneProjectsTo>${project.build.directory}/it</cloneProjectsTo>
+                    <cloneClean>true</cloneClean>
+                    <postBuildHookScript>verify</postBuildHookScript>
+                    <addTestClassPath>true</addTestClassPath>
+                    <skipInvocation>${skipTests}</skipInvocation>
+                    <streamLogs>true</streamLogs>
```

Comment:
I don't think we need this one, right?

Reply:
I have a very weird error when trying to remove this configuration so I prefer keep like this